### ISSUE TITLE
Quality of code

### ISSUE THREAD

**Issue by odirleiborgert** [odirleiborgert/borgert-cms#28]
Improving the quality of code

Refactor 
'app/Libraries/UploadHandler.php'

**Comment by odirleiborgert** [comment]
Refactor 
'app/Libraries/UploadHandler.php'

**Comment by Spudley** [comment]
Hi.

I've got a bit of time available, so I'm going to give this a go.

However, having looked at the code, there's probably more work there than I have time for. I'll do what I can, but there will probably be scope for further refactoring after I'm done.

**Comment by odirleiborgert** [comment]
Thanks @Spudley

**Comment by odirleiborgert** [comment]
Hi @Spudley 
Do not worry, I'll be waiting for your pull request.
And thank you for collaborating with the project.

**Comment by odirleiborgert** [comment]
Hi @Spudley 
I believe that this link you can see some improvements to be made.

https://scrutinizer-ci.com/g/odirleiborgert/borgert-cms/code-structure/master/class/App%5CLibraries%5CUploadHandler

**Comment by Spudley** [comment]
Hi. As promised I have submitted the pull request for the refactoring work I did previously.

(apparently the problem I had with Github trying to commit it on Friday was due to a massive DDoS attack that also took down Twitter and a bunch of other big sites and was big enough to get mentioned on the main news on TV that night. Hopefully they weren't just doing it to stop me submitting these changes! hehehe)
